Crash report 2.7.3 - IllegalArgumentException in SetupBlog

```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:200)
at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:274)
at java.util.concurrent.FutureTask.setException(FutureTask.java:125)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:308)
at java.util.concurrent.FutureTask.run(FutureTask.java:138)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1088)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:581)
at java.lang.Thread.run(Thread.java:1019)
Caused by: java.lang.IllegalArgumentException: Host name may not be null
at org.apache.http.HttpHost.<init>(HttpHost.java:83)
at org.apache.http.impl.client.AbstractHttpClient.determineTarget(AbstractHttpClient.java:497)
at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:487)
at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:465)
at org.xmlrpc.android.XMLRPCClient$Caller.callXMLRPC(XMLRPCClient.java:467)
at org.xmlrpc.android.XMLRPCClient$Caller.access$000(XMLRPCClient.java:395)
at org.xmlrpc.android.XMLRPCClient.call(XMLRPCClient.java:219)
at org.xmlrpc.android.XMLRPCClient.call(XMLRPCClient.java:214)
at org.wordpress.android.ui.accounts.SetupBlog.getmXmlrpcByUserEnteredPath(SetupBlog.java:211)
at org.wordpress.android.ui.accounts.SetupBlog.getSelfHostedXmlrpcUrl(SetupBlog.java:292)
at org.wordpress.android.ui.accounts.SetupBlog.getBlogList(SetupBlog.java:104)
at org.wordpress.android.ui.accounts.WelcomeFragmentSignIn$SetupBlogTask.doInBackground(WelcomeFragmentSignIn.java:434)
at org.wordpress.android.ui.accounts.WelcomeFragmentSignIn$SetupBlogTask.doInBackground(WelcomeFragmentSignIn.java:404)
at android.os.AsyncTask$2.call(AsyncTask.java:185)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:306)
... 4 more
```

--- a/src/org/wordpress/android/ui/accounts/SetupBlog.java
+++ b/src/org/wordpress/android/ui/accounts/SetupBlog.java
@@ -197,7 +197,7 @@ public class SetupBlog {
 
     private String getmXmlrpcByUserEnteredPath(String baseUrl) {
         String xmlRpcUrl = null;
-        if (!UrlUtils.isValidUrl(baseUrl)) {
+        if (!UrlUtils.isValidUrlAndHostNotNull(baseUrl)) {
             AppLog.e(T.NUX, "invalid URL: " + baseUrl);
             mErrorMsgId = R.string.invalid_url_message;
             return null;

--- a/src/org/wordpress/android/util/UrlUtils.java
+++ b/src/org/wordpress/android/util/UrlUtils.java
@@ -151,7 +151,7 @@ public class UrlUtils {
     /**
      * returns false if the url is not valid or if the url host is null, else true
      */
-    public static boolean isValidUrl(String url) {
+    public static boolean isValidUrlAndHostNotNull(String url) {
         try {
             URI uri = URI.create(url);
             if (uri.getHost() == null) {

